model/account_metadata.go

```diff
@@ -9,6 +9,7 @@ import "encoding/json"
 // AccountMetadata is the provider metadata stored in a model.Account.
 type AccountMetadata struct {
 	Provision bool
+	Subnet    string
```

Comment:
I wonder if we could avoid storing Subnet in `Account` as we already assign AccountID to Subnet entity which results in the same data being duplicated in two different places. Because of this we need to remember to always keep it in sync and it is easy to forget that after some time away from code while adding new functionalities etc 😅 

1/5 on that, from my limited knowledge about Genesis is a little hard to say which approach is better.

Reply:
As discussed we actually need this for Terraform deployments. Maybe sth we can improve later.